site/styles/_search.scss

```diff
@@ -62,5 +62,5 @@
 
 .s72-form-search {
   margin: 0 auto 1rem;
-  max-width: map-get($container-max-widths, md);
+  max-width: 400px;
```

Comment:
`400px` seems small on desktop. It was the width of the `md` breakpoint at `768px` - meet halfway-ish at `600px`?

Reply:
According to [this](https://www.black.ninja/7-usability-findings-and-recommendations-to-improve-your-website-2/#:~:text=4.-,THE,-IDEAL%20SEARCH%20BOX) article the ideal search box is 27 characters wide. 

Having 400px wide, it is enough space to accommodate 39 characters.

Reply:
You are quoting a website using iPhone 4 screens - doesn't inspire confidence, but even they say "In general, search boxes are better too wide than too short".

As a counter-example Google allows 60 characters at 533px wide.

![image](https://user-images.githubusercontent.com/2253814/155903505-3091e250-3255-4652-98be-46d653483a95.png)

Reply:
Sorry I sent the wrong article, [this](https://www.yellowslice.in/bed/best-practices-for-designing-search-box-and-filter-design/#:~:text=Ensure%20that%20the%20Input%20Field%20is%20Aptly%20Sized) article is also referring to 27 characters wide, which applies for desktop too.

Please correct me if I am wrong, but the user will search for a film name or crew member. Shouldn't be a concern then, because it won't be a long phrase/question in which the user needs to check what they are writing. 

But anyway, I increased the size to 460px wide now to accommodate 46 characters. Let me know what you think :)

Reply:
That article doesn't mention a source, but I expect they will both be referencing the Nielsen study that was itself referenced in [2009 by Smashing Magazine](https://www.smashingmagazine.com/2009/09/10-useful-usability-findings-and-guidelines/#6-the-ideal-search-box-is-27-characters-wide).

We don't know what they will search for (we could maybe do something with Google Analytics or start saving them for analysis), but yes my guess is low-character count searches of film/cast/crew names. However my initial comment was not about character counts, it was the reduced visual prominence - the design seemed cramped for no reason.